src/test/java/user/Entity/DAO/UserDAOImplTest.java

```diff
@@ -0,0 +1,109 @@
+package user.Entity.DAO;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import user.Entity.User;
+
+import javax.sql.DataSource;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+/**
+ * Created by user on 22.05.2016.
+ */
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes=UserDAOImplConf.class)
+public class UserDAOImplTest {
+    @Autowired
+    private UserDAO my;
+
+    @Configuration
+    public static class Conf {
+        @Bean
+        DataSource dataSource() {
```

Comment:
move this dataSource in your real conf, + add annotation @Profile("test") to this bean